build.proj

```diff
@@ -408,25 +250,69 @@
   </Target>
 
   <Target Name="TestFailureErrorMessage">
-    <Error Text="Test failures occured, check the files in artifacts/Test" />
+    <Error Text="Test failures occured, check the files in artifacts/TestResults" />
   </Target>
 
-  <Target Name="BinScopeCheck">
-    <Exec Command="$(PowerShellCoreCommandPrefix) &quot;$(MSBuildProjectDirectory)\tools\CheckBinScope.ps1 -BuildConfig $(Configuration) -CIToolsPath $(CIToolsPath)&quot;" ContinueOnError="False" />
-    <OnError ExecuteTargets="BinScopeErrorMessage" />
-  </Target>
+  <Target Name="CodeSignBinaries" AfterTargets="Build" Condition="'$(CodeSign)' == 'true'">
+    <Message Text="====> Executing CodeSignBinaries Target..." Importance="high" />
+    <Message Text="----> Dlls signing section" Importance="high" />
+
+    <!-- Azure -->
+    <ItemGroup Condition="'$(Scope)' == 'All' or '$(Scope)' == 'Latest' or '$(Scope)' == 'Netcore'">
+      <DelaySignedAssembliesToSign Include="
+$(PackageDirectory)/$(Configuration)/**/Microsoft*Azure*PowerShell*Cmdlets*.dll;
```

Comment:
It would be really super-nice if we could get these from the Output proeprty of the csproj files list above.  Oh well, next ADO update

Reply:
That's a large engineering effort that, at this point, I don't think is worth the effort since the generator has a completely different project structure. If I was able to do this in December, I think it would have made sense. 😢